easy/admin/field.py

```diff
@@ -85,6 +88,39 @@ def render(self, obj):
         return self.default
 
 
+class ManyToManyAdminField(SimpleAdminField):
+    """Renders a ManyToManyField as links."""
+
+    def __init__(self, attr, display=None, short_description=None, admin_order_field=None, default=None):
```

Comment:
I think will be great to add a template path. to help on customization from user.

then you check
``` python
if template_path:
    # use template
else:
    # use default way
```